paimon-common/src/main/java/org/apache/paimon/CoreOptions.java

```diff
@@ -459,7 +459,9 @@ public class CoreOptions implements Serializable {
                                             text(
                                                     "3. \"millis-to-micro\": Pads the sequence field that indicates time with precision of milli-second to micro-second."),
                                             text(
-                                                    "4. Composite pattern: for example, \"second-to-micro,row-kind-flag\"."))
+                                                    "4. \"inc-seq\": Pads the sequence field with auto increase sequence."),
+                                            text(
+                                                    "5. Composite pattern: for example, \"second-to-micro,row-kind-flag\"."))
```

Comment:
`inc-seq, row-kind-flag` is supported? I think maybe this should be most popular composite pattern

Reply:
`inc-seq` + `row-kind-flag` will further reduce the accuracy, will there be a situation where +U come first, and then -U come, at the same second?